cypress/e2e/nfts.test.ts

```diff
@@ -5,8 +5,9 @@ const BONSAI_COLLECTION_ADDRESS = '0xec9c519d49856fd2f8133a0741b4dbe002ce211b'
 
 describe('Testing nfts', () => {
   beforeEach(() => {
-    cy.visit('/')
-    cy.get(getTestSelector('FiatOnrampAnnouncement-close')).first().click()
+    cy.visit('/').then(() => {
+      cy.get(getTestSelector('FiatOnrampAnnouncement-close')).first().click()
+    })
```

Comment:
gah, surprised this is necessary 🙈

Reply:
i'm not sure if it's necessary in this case but i think it's good practice? especially in the other test, where we want to make sure the page loads before looking for specific elements